packages/design-system-tailwind-preset/src/typography.test.ts

```diff
@@ -11,14 +11,6 @@ describe('Typography', () => {
 
   // Ignored CSS variables
   const ignoreList: string[] = [
-    '--typography-s-heading-sm-regular-font-size',
-    '--typography-s-heading-sm-regular-line-height',
-    '--typography-s-heading-sm-regular-font-weight',
-    '--typography-s-heading-sm-regular-letter-spacing',
-    '--typography-l-heading-sm-regular-font-size',
-    '--typography-l-heading-sm-regular-line-height',
-    '--typography-l-heading-sm-regular-font-weight',
-    '--typography-l-heading-sm-regular-letter-spacing',
```

Comment:
Ensured `s-heading-sm-regular` is not used in the codebase. Only instances are in changelog and migration doc

<img width="491" alt="Screenshot 2025-05-25 at 2 17 54 PM" src="https://github.com/user-attachments/assets/57ad734a-11c3-473f-8dc0-8a17ce65ed98" />